src/Cardano/Wallet/Network/HttpBridge.hs

```diff
@@ -83,34 +83,38 @@ newNetworkLayer network port = mkNetworkLayer <$> newHttpBridge network port
 -- | Retrieve a chunk of blocks from cardano-http-bridge.
 --
 -- It will either return:
+--
 -- - an epoch pack's worth of blocks (those after the given starting slot); or
--- - all of the unstable blocks after the starting slot, if any.
+-- - all of the unstable blocks after (exclusively) the starting slot, if any.
+--
+-- Note that, starting from the `SlotId 0 0`, we'll never get the first block of
+-- the chain through this method.
 rbNextBlocks
     :: Monad m
     => HttpBridge m -- ^ http-bridge API
     -> SlotId -- ^ Starting point
     -> ExceptT ErrNetworkUnreachable m [Block]
 rbNextBlocks network start = maybeTip (getNetworkTip network) >>= \case
     Just (tipHash, tipHdr) -> do
-        epochBlocks <- nextStableEpoch (epochNumber start)
+        epochBlocks <-
+            if slotNumber start >= 21599
```

Comment:
every epoch has `21599` slots, right? Maybe it would be good to inject this value for the sake of testing boundaries without relying on mainnet